client/src/app/shared/components/choice-dialog/choice-dialog.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ChoiceDialogComponent } from './choice-dialog.component';
```

Comment:
to get the thests up and running, you need to import the e2e imports module.
Also use prettier in the TS files whenever you can. Indentation is still the angular basic style